src/validation.cpp

```diff
@@ -624,11 +624,6 @@ static bool AcceptToMemoryPoolWorker(const CChainParams& chainparams, CTxMemPool
             return state.DoS(0, false, REJECT_INSUFFICIENTFEE, "mempool min fee not met", false, strprintf("%d < %d", nFees, mempoolRejectFee));
         }
 
-        // No transactions are allowed below minRelayTxFee except from disconnected blocks
```

Comment:
Maybe preserve this comment (move it up by mempoolRejectFee check).

Also maybe drop the `mempoolRejectFee > 0` check above which seems impossible to hit now (and confusing & unnecessary previously).

Reply:
Thanks for the review, fixed